tests/Application/src/Repository/ProductRepository.php

```diff
@@ -2,8 +2,10 @@
 
 declare(strict_types=1);
 
-namespace StefanDoorn\SyliusSeoUrlPlugin\Repository;
+namespace Tests\Application\StefanDoorn\SyliusSeoUrlPlugin\Repository;
```

Comment:
I added this one here so people could use it directly if they don't want to extend the repository themselves, see README. WDYT?

Reply:
Seems reasonable, but I think usually people use more plugin than one and at the end, have all models, repositories etc. overridden in their `src/` folder anyway... But I don't insist, I think it's a matter of taste, the README explains the approach properly 👌 I'll remove that